share/jupyter/voila/templates/default/nbconvert_templates/voila.tpl

```diff
@@ -15,6 +15,23 @@
     <link rel="stylesheet" type="text/css" href="{{resources.base_url}}voila/static/theme-light.css">
 {% endif %}
 
+<style>
+  .spinner {
+      -webkit-animation: rotation 2s infinite linear;
+    -moz-animation: rotation 2s infinite linear;
+    -o-animation: rotation 2s infinite linear;
+    animation: rotation 2s infinite linear;
+      transform-origin: 50% 50%;
+      -webkit-transform-origin: 50% 50%;
+      -moz-transform-origin: 50% 50%;
```

Comment:
I am not sure you still need the `-moz`, `-o` and `-webkit` versions of css properties.

Reply:
Fixed :) Thanks